fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -2317,8 +2317,8 @@ private void validateReferenceParams(List<QueryParameter> chainQueryParameters,
     /**
      * Creates a bundle that will hold the results of a history operation.
      *
-     * @param resources
-     *            the list of resources to include in the bundle
+     * @param resourcesResults
+     *            the list of resource results to include in the bundle, sorted with the more recent changes first
```

Comment:
This depends on the _sort request, surely?

Reply:
oh, did you add _sort support for instance-level history as well?

Reply:
oh, I thought this bit was common for the history calls...in this case the sorting is fixed according to the SQL_HISTORY_FROM_DATETIME query in ResourceDAOImpl